python/tests/ecl/test_fk_user_data.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#  Copyright (C) 2017  Statoil ASA, Norway. 
+#   
+#  The file 'test_ecl_cell_containment.py' is part of ERT - Ensemble based Reservoir Tool.
+#   
+#  ERT is free software: you can redistribute it and/or modify 
+#  it under the terms of the GNU General Public License as published by 
+#  the Free Software Foundation, either version 3 of the License, or 
+#  (at your option) any later version. 
+#   
+#  ERT is distributed in the hope that it will be useful, but WITHOUT ANY 
+#  WARRANTY; without even the implied warranty of MERCHANTABILITY or 
+#  FITNESS FOR A PARTICULAR PURPOSE.   
+#   
+#  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html> 
+#  for more details.
+
+from ecl.ecl import EclGrid
+from ecl.test import ExtendedTestCase, TestAreaContext
+
+class FKTest(ExtendedTestCase):
+
+    def test_cell_containment(self):
+
+        grid_location = "local/ECLIPSE/faarikaal/faarikaal%d.EGRID"
+        well_location = "local/ECLIPSE/faarikaal/faarikaal%d.txt"
+
+        for i in range(1, 8):
+            grid_file = self.createTestPath(grid_location % i)
+            well_file = self.createTestPath(well_location % i)
+
+            grid = EclGrid.loadFromFile(grid_file)
```

Comment:
You're absolutely right! But then I think we should make EclGrid.loadFromFile private and deprecate the current public version of the function. This to be in line with the "only one obvious way to do it" Python mantra..